tests/ballerina-unit-test/src/test/resources/test-src/expressions/varref/record-variable-reference-negative.bal

```diff
@@ -121,3 +121,12 @@ function testUnknownFields() {
     any unknown2;
     {name, married, age: {age, format, unknown1}, unknown2} = p; // unknown fields
 }
+
+type UserData1 record {
+    *Data;
```

Comment:
Shall we add this test to `ClosedRecordTypeReferenceTest` and `OpenRecordTypeReferenceTest`?

Reply:
Resolved with https://github.com/ballerina-platform/ballerina-lang/pull/12628/commits/9e563442150aeb314adcb739451841f0ca4f2f49.